packages/react/src/combobox/utils.test.ts

```diff
@@ -39,4 +39,22 @@ describe('splitLabel', () => {
 		expect(splitLabel(option, 'New ')).toEqual(['New ', 'Zealand']);
 		expect(splitLabel(option, 'New Z')).toEqual(['New Z', 'ealand']);
 	});
+
+	test('it should split a label with special characters correctly', () => {
```

Comment:
There are lots of other tests in this file for this function that are still working as expected - these are new tests to fix this edge case